src/server.rs

```diff
@@ -41,7 +42,7 @@ pub(crate) async fn handle_logs(
         }
     };
 
-    let body = match body::to_bytes(body)
+    let body = match body::to_bytes(body, usize::MAX)
```

Comment:
does this mean we might possibly attempt to allocate a few exabytes of RAM?

Reply:
theoretically yes, but: 
- this was also the default behaviour in `hyper::body::to_bytes` 
- we only do this _after_ authentication, so we already know the request is valid.